upgrade-legacy-d7.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+echo "This script will convert your Drupal 7 project to be compatible with composer, lando and Silta."
+echo "Make sure you are in the git root of your project."
+
+echo "Checking dependencies"
+git --version
+composer --version
+lando version
+json_pp -v
+
+# Use the current directory as the project name
+CURRENT_PATH=`pwd`
+PROJECT_NAME=`basename $CURRENT_PATH`
+
+# Update build.sh
+curl -s https://raw.githubusercontent.com/wunderio/WunderTools/drupal7/build.sh > build.sh
+curl -s https://raw.githubusercontent.com/wunderio/WunderTools/drupal7/drupal/build.sh > drupal/build.sh
+
+# Update site.yml
+mv drupal/conf/site.yml drupal/conf/site.yml.old
+curl -s https://raw.githubusercontent.com/wunderio/WunderTools/drupal7/drupal/conf/site.yml > drupal/conf/site.yml
+
+# Download lando configuration file, renaming the project on the way.
+curl -s https://raw.githubusercontent.com/wunderio/WunderTools/drupal7/drupal/.lando.yml | sed "s/wundertools/$PROJECT_NAME/" > drupal/.lando.yml
+
+# Download the base composer file.
```

Comment:
Should these be taken from wundertools/drupal since they have been modified and made to work properly with wundertools structure?

Reply:
Yes, now that the examples are moved out of the composer.json this makes a lot of sense.